docs/monoDrive_home/Vehicle-PhysX.md

```diff
@@ -0,0 +1,51 @@
+# PhysX Vehicles
+
+Unreal Engine uses PhysX for the vehicle physics model which monoDrive's Traffic vehicles inherit from. Through Unreal Engine, users will be able to customize the mechanical settings of PhysX vehicles in the monoDrive Scenario Editor and Simulator.
+
+To see details of the PhysX model, see [PhysX documentation](https://docs.nvidia.com/gameworks/content/gameworkslibrary/physx/guide/Manual/Vehicles.html).
+
+## Suspension, Wheel, and Tire Configuration
+
+To customize the suspension, wheel or tire properties, create a wheel class that is a child blueprint child of the VehicleAIWheelFront(and Rear). 
+    
+1. Navigate to the VehicleAIWheel either directly in the Content browser or by selecting the magnifying glass under Vehicle Setup -> Wheel Setups in any of the VehicleMovement components attached to any of the blueprint vehicles in the Vehicles folder.
+
+    <p class="img_container">
+    <img class="wide_img" src="../img/wheel_config1.png" />
+    </p>
+
+1. After sub-classing the wheel class, the wheel with settings can be customized. This includes all the suspension settings in PhysX, tire stiffness and friction properties, mass, collision geometry, and more.
+
+    <p class="img_container">
+    <img class="wide_img" src="../img/wheel_config2.png" />
+    </p>
+
+To set a custom tire configuration which contains the tires friction parameter (note that friction is also set on the materials in the environment and scaled by this number), right click in the Content Browser, and then select Miscellaneous -> Data Asset and search for TireConfig. This will create a custom tire configuration that can then be selected in the wheel blueprint. Note that the default value is 1 which means that all friction properties will be derived by the environment’s materials. Changing this property is only recommended for advanced users, the friction is multiplicative with the environment surface’s frictional value.
```

Comment:
To set a custom tire configuration that contains the tires friction parameter (NOTE: that friction is also set on the materials in the environment and scaled by this number), right click in the Content Browser, select Miscellaneous -> Data Asset, and search for TireConfig. This will create a custom tire configuration that can then be selected in the wheel blueprint. Note that the default value is 1 meaning all friction properties will be derived by the environment’s materials. The friction is multiplicative with the environment surface’s frictional value, so be cautious when changing this value because this combination can make the wheel and vehicle's physics unstable and unpredictable if not set correctly.